### PR TITLE
Controller tests for secrets and Strong parameters

### PR DESCRIPTION
**Pull request by annaswims**

### CODE REVIEW

**Comment by iangreenleaf** [comment]
Nice! Looks really good. :+1:

File: app/controllers/secrets_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class SecretsController < ApplicationController
-  before_filter :require_user, :load_autofingers, except: [:new, :create]
+  before_filter :require_user, :load_autofingers
```
Comment (iangreenleaf) [inline app/controllers/secrets_controller.rb]: I think the current code allows people to post secrets without being logged in (for anyone worried about being tracked). We'll probably want to keep that capability?

**Comment by annaswims** [comment]
HA!  I re-read the github issues and saw that I made a few mistakes related to who can do what. no need to review until I fix it up.

**Comment by annaswims** [comment]
You'll notice that `secret.to_s` is displayed when you run rpsec, It's because `display` is both a ruby method on object and the name of a column in `secret`.

**Comment by annaswims** [comment]
ok, should be fixed now.

**Pull request merged**